src/components/VForm.tsx

```diff
@@ -0,0 +1,11 @@
+import { ofType } from 'vue-tsx-support';
+import { VForm } from 'vuetify/lib';
+
+export default ofType<Props>().convert(VForm as any);
+
+type Props = {
+  ref?: string;
+  'v-model'?: any;
```

Comment:
I'm not sure if this can be `boolean`

Reply:
Hey @Shinigami92 I'm not sure if we should be putting `v-model` on Props here nor `ref`.

Reply:
I thought this could be helpfull for autocompletion

Reply:
I think `vue-tsx-support` expose `ref` but not `v-model` since with a vue thing not tsx specific.

Reply:
Should I remove this line or can you cherry-pick some things?

Reply:
yes remove it.